src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    private String name;
+    private boolean done;
```

Comment:
Perhaps a change to `isDone` for the variable to more clearly communicate that it is a boolean variable?

Reply:
Seconding this!